FinSearchUnified/Tests/Helper/UrlBuilderTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace FinSearchUnified\Tests\Helper;
+
+use FinSearchUnified\Helper\UrlBuilder;
+use Shopware\Bundle\SearchBundle\Criteria;
+use Shopware\Components\Test\Plugin\TestCase;
+use \Zend_Http_Client;
```

Comment:
The leading backslashes shouldn't actually be necessary here.

Reply:
Done!